test/run_test_apache.sh

```diff
@@ -16,8 +16,8 @@ function random-string() {
 source ./utils.sh
 
 # check argument number
-if [ "$#" -lt 2 ] || [ "$#" -gt 4 ] ; then
-    error_exit "Usage: ./run_test.sh <version> <path to apache config folder> <pressure> <ssl>.  Aborting."
+if [ "$#" -lt 2 ] || [ "$#" -gt 6 ] ; then
```

Comment:
I'm not sure you want to do it in this PR, but it would be great to improve the parameters hacking to a more recent and convenient parameter handling. Take a look at https://github.com/RenatGilmanov/shell-script-template for templates.